### PR TITLE
Wish list

### PR DESCRIPTION
**Pull request by srishtigautam21**
Feat: add to wishList functionality to my WishList Page
**Files to check:**
WishListCard
WishListContext
WishList

### CODE REVIEW

File: src/component/ProductsFilter/ProductsFilter.jsx [hunk 1]
```diff
@@ -15,9 +15,9 @@ const ProductsFilter = () => {
             id='lowToHigh'
             name='radio-lowToHigh'
             type='radio'
-            onClick={() =>
-              dispatch({ type: "SORTING", payload: "low to high" })
-            }
+            // onClick={() =>
+            //   dispatch({ type: "SORTING", payload: "low to high" })
+            // }
```
Comment (itsmebipul00) [inline src/component/ProductsFilter/ProductsFilter.jsx]: Comments could be removed

File: src/component/ProductsFilter/ProductsFilter.jsx [hunk 2]
```diff
@@ -27,9 +27,9 @@ const ProductsFilter = () => {
             id='highToLow'
             name='radio-highToLow'
             type='radio'
-            onClick={() =>
-              dispatch({ type: "SORTING", payload: "high to low" })
-            }
+            // onClick={() =>
```
Comment (itsmebipul00) [inline src/component/ProductsFilter/ProductsFilter.jsx]: Here also comments could be removed

**Review by rutu14 — APPROVED** [review]
The structure looks good to merge.

File: src/component/Card/WishListCard.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import "./wishListCard.css";
+import "./card.css";
+import "../index";
+import { useWishList } from "../Context/WishListContext/WishListContext";
+
+const WishListCard = ({ wishlist }) => {
```
Comment (itsmebipul00) [inline src/component/Card/WishListCard.jsx:6]: Suggestion: You could destructure wishlistem and wishListCount here itself(line:6)

File: src/component/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -44,20 +46,30 @@ const Navbar = () => {
...
             <Link className='link ecom-link-color' to='/wishlist'>
-              <HeartIcon className='nav-icons' />
-              <p>Wishlist</p>
+              <div className='ecom-badge-wrapper'>
+                <HeartIcon className='nav-icons' />
+                <div className='badge icon-over-badge'>
+                  {wishlist.wishListCount}
+                </div>
+              </div>
+              {/* <HeartIcon className='nav-icons' /> */}
```
Comment (itsmebipul00) [inline src/component/Navbar/Navbar.jsx]: Comments could be removed

**Review by itsmebipul00 — APPROVED** [review]
Code is good, good use of semantics, comments could be removed, good to merge

File: src/component/Card/WishListCard.jsx (hunk 3 above)
Comment (srishtigautam21) [reply src/component/Card/WishListCard.jsx:6]: If i am doing that my code is breaking thats why i destructed it outside

**Pull request merged**